ci/Dockerfile

```diff
@@ -205,6 +205,8 @@ RUN export DEAL_II_HASH=7ac7850030de7a2027679099073f00f59a076ac4 && \
         -DTRILINOS_DIR=${TRILINOS_DIR} \
         -DDEAL_II_WITH_ARPACK=ON \
         -DARPACK_DIR=${ARPACK_DIR} \
+        -DDEAL_II_WITH_CUDA=ON \
+        -DDEAL_II_WITH_CXX14=OFF \
```

Comment:
Is turning C++14 off required for Cuda?

Reply:
Yes exactly. deal.II does not support compiling the CUDA code with a different C++ standard than the rest of the library.